SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs

```diff
@@ -11,20 +14,16 @@ public partial class Crashes : IAppCenterService
     {
         public string ServiceName => "Crashes";
 
+        /// <inheritdoc />
```

Comment:
Does this actually work for you? I was never able to get it working for me

Reply:
It does not do anything visible except removing the warning